ts/parts-more/Polar.ts

```diff
@@ -793,6 +1039,39 @@ H.addEvent(H.Chart, 'afterDrawChartBox', function (
     });
 });
 
+H.addEvent(H.Series, 'afterInit', function (
+    this: Highcharts.Series
+): void {
+    var chart = this.chart;
+
+    // Add flags that identifies radial inverted series
+    if (chart.inverted && chart.polar) {
+        this.isRadialSeries = true;
+        if (this instanceof seriesTypes.column) {
+            this.isRadialBar = true;
+        }
+    }
+});
+
+H.addEvent(H.Series, 'afterRender', function (
```

Comment:
We don't use `point.complete` anymore, we can get rid of this now, right? Also make sure it is removed from the TypeScript interface.

Reply:
Got rid of the event, thanks!